src/python/pants/backend/python/target_types.py

```diff
@@ -241,7 +241,9 @@ class PexPlatformsField(StringSequenceField):
     help = (
         "The platforms the built PEX should be compatible with.\n\nThis defaults to the current "
         "platform, but can be overridden to different platforms. You can give a list of multiple "
```

Comment:
Hm, and I think actually the best place to talk about building on other platforms requiring wheels is here:

> This defaults to the current platform, but can be overridden to different platforms. When building for a foreign platform, there must be built wheels for those platforms available, rather than sdists.